docs/tech/attributes.md

```diff
@@ -5,7 +5,7 @@ Attributes values are encrypted using the keys defined below (section Attributes
 
 ## Attributes Keys
 
-The attributes are encrypted using a key derived from a hierarchy called PreKey tree. This approach consists in deriving keys from a parent key, allowing the parent key owner, get access to derived keys. This way keys used to encrypt attributes are different among attributes and TCerts avoiding linkability while allowing an authorized auditor who owns a parent key to derive the keys in the lower levels.  
+The attributes are encrypted using a key derived from a hierarchy called PreKey tree. This approach consists in deriving keys from a parent key, allowing the parent key owner, get access to derived keys. This way keys used to encrypt attributes are different among attributes and TCerts avoiding linkability while allowing an authorized auditor who owns a parent key to derive the keys in the lower levels.
```

Comment:
Yes, that's true - sorry, I'm on a roll here ;-) Please ignore it and I'll update separately. We'd have better history this way.